src/main/java/edu/cornell/kfs/concur/rest/application/ConcurTestConnectionApplication.java

```diff
@@ -0,0 +1,31 @@
+package edu.cornell.kfs.concur.rest.application;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import edu.cornell.kfs.concur.service.impl.ConcurTestConnectionWebServiceImpl;
+
+@ApplicationPath("/systems/v1.0")
```

Comment:
Concur expects this endpoint to be: {servername}/system/v1.0/testconnection

I am not sure this that can be customized. I will double check with Concur. thank you